src/sql/base/test/browser/ui/scrollableView/scrollableView.test.ts

```diff
@@ -63,128 +63,141 @@ suite.skip('ScrollableView', () => { // TODO chgagnon Fix these tests
 		scrollableView.dispose();
 	});
 
-	test('adds and removes views correctly', async () => {
-		const view1 = new TestView(20, 20);
-		const view2 = new TestView(20, 20);
-		const view3 = new TestView(20, 20);
-		const scrollableView = new ScrollableView(container);
+	test('adds and removes views correctly', async (): Promise<void> => {
+		(async () => {
```

Comment:
I don't think this is going to help - unless I'm mistaken these tests will pass but that's because they're essentially not running. By wrapping them in an async function that you don't await on the test will immediately return and thus "pass" even if the wrapped test fails. 

What were you trying to accomplish with this approach?

Reply:
Huh, or maybe I'm wrong? The tests do appear to be running and failing in the CI build...which is definitely not what I would have expected. Do you have context here on what's making that work?

Reply:
Looking at it closer - the errors are being caught by the unhandled error check (`should not have unexpected errors:`)

So what I said does look like it's happening. The tests are passing because they kick off an async function and immediately return 

![image](https://user-images.githubusercontent.com/28519865/130293847-663ab858-a2cb-44d3-b517-ce949a59d6da.png)

I'm going to make an educated guess that the tests are passing without errors on Linux because we set up a virtual frame buffer to simulate a screen (https://github.com/microsoft/azuredatastudio/blob/main/build/azure-pipelines/linux/sql-product-build-linux.yml#L116). Windows & Mac don't have any such thing set up and the lab machines are run without a UI context and so the animation stuff is never triggering. 

So either have to:

1. Figure out if we can do a similar thing for Mac/Windows
2. Figure out how we can get the animation stuff to tick even without an actual UI context available